app/controllers/chargeback_assignment_controller.rb

```diff
@@ -13,7 +13,7 @@ def self.table_name
 
   def index
     @title = _("Chargeback Assignments")
-    @tabform = "Compute"
+    @tabform = params[:tab] && ChargebackRate::VALID_CB_RATE_TYPES.include?(params[:tab]) ? params[:tab] : "Compute"
```

Comment:
You don't need to check for the existence:
```suggestion
    @tabform = ChargebackRate::VALID_CB_RATE_TYPES.include?(params[:tab]) ? params[:tab] : "Compute"
```

Reply:
@skateman was trying to avoid lookup if `params[:tab]` was not even passed in, but i will make the requested change